packages/block-library/src/html/editor.scss

```diff
@@ -23,8 +14,32 @@
 			font-size: $default-font-size;
 		}
 
-		&:focus {
-			box-shadow: none;
+		&:focus::after {
```

Comment:
This is copied from [here](https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/block-list/style.scss#L72). Any ideas for how to simplify this and avoid unnecessary repetition?